core/server/src/test/java/alluxio/worker/block/AbstractClientTest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.worker.block;
+
+import alluxio.AbstractClient;
+import alluxio.Constants;
+import alluxio.exception.ExceptionMessage;
+import alluxio.thrift.AlluxioService;
+import alluxio.thrift.AlluxioService.Client;
+import alluxio.thrift.BlockMasterClientService;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import java.io.IOException;
+import java.net.InetSocketAddress;
+
+/**
+ * Unit tests for {@link AbstractClient}.
+ */
+public final class AbstractClientTest {
+  private static final String SERVICE_NAME = "Test Service Name";
+  private static final int CLIENT_SERVICE_VERSION = 1;
+
+  private final class TestClient extends AbstractClient {
+    private TestClient() {
+      super(Mockito.mock(InetSocketAddress.class), "");
+    }
+
+    @Override
+    protected Client getClient() {
+      return null;
+    }
+
+    @Override
+    protected String getServiceName() {
+      return SERVICE_NAME;
+    }
+
+    @Override
+    protected long getServiceVersion() {
+      return CLIENT_SERVICE_VERSION;
+    }
+
+    public void checkVersion(AlluxioService.Client thriftClient) throws IOException {
```

Comment:
Good catch, I initially copied from the BlockMasterClientTest and forgot to replace everything specific to BlockMaster. It should all be updated now (I followed your suggestion)